### ISSUE TITLE
batch normalization question

### ISSUE THREAD

**Issue by HotMarijke** [tiny-dnn/tiny-dnn#292]
Why doesn't the batch normalization layer apply scaling and shifting like in the original paper? 

thanks,
Marijke

**Comment by nyanp** [comment]
@HotMarijke 
tiny-dnn's batch normalization is ported from [caffe's one](https://github.com/BVLC/caffe/blob/master/src/caffe/layers/batch_norm_layer.cpp), which don't have scale and bias operation. We already have scaling and shifting layer named ```linear_layer```, and if we can train it's scale and bias, we can emulate the original paper by combine them.

**Comment by HotMarijke** [comment]
Okay, now I understand!
Thanks for this great project!

Marijke
